cli-helpers/src/commands/create-user/index.ts

```diff
@@ -145,6 +145,14 @@ export default class CreateUser extends Command {
                         Name: "family_name",
                         Value: lastName,
                     },
+                    {
+                        Name: "email_verified",
+                        Value: "true",
+                    },
+                    {
```

Comment:
not receiving an email when entering an existing address with an account (checked my junk mail also)?

Reply:
That will be because currently any existing users do not have verified emails. I have just manually changed this in the AWS console for all users in the sandbox, test, pre-prod and prod accounts. Any future users that are created their emails will be automatically verified as part of the createUser function.